File: src/test/java/com/sidgate/CardTest.java
package com.sidgate;

import com.sidgate.deck.Card;
import com.sidgate.deck.Rank;
import com.sidgate.deck.Suit;
import org.junit.Assert;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class CardTest {
	@Test
	public void testCreateCard() {
		Card card = new Card("H5");
		Assert.assertEquals(Suit.HEART, card.suit());
		Assert.assertEquals(Rank.FIVE, card.rank());
	}
}

File: src/test/kotlin/sidgate/CardTest.kt
package sidgate

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test
import sidgate.deck.Card
import sidgate.deck.Rank
import sidgate.deck.Suit

class CardTest {
    @Test
    fun testCreateCard() {
        val card = Card("H5")
        assertEquals(Suit.HEART, card.suit)
        assertEquals(Rank.FIVE, card.rank)
    }
}